packages/neutrino/bin/build.js

```diff
@@ -7,10 +7,7 @@ module.exports = (middleware, options) => {
   return build(middleware, options)
     .fork((errors) => {
       spinner.fail('Building project failed');
-      errors.forEach((err) => {
-        console.error(err.stack || err);
-        err.details && console.error(err.details);
-      });
+      errors.forEach(err => console.error(err));
```

Comment:
the rambda stuff seems not worth it IMO, adds an extra layer of indirection for a lot of these things, making it harder to reason about if you're not familiar with them all. Some might be more obvious but if I saw `unary()` I'd just be like huh wtf is that